packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -341,7 +341,7 @@ typedef NestedScrollViewHeaderSliversBuilder = List<Widget> Function(BuildContex
 ///
 /// ### Stretching [SliverAppBar]s
 ///
-// TODO(Piinks): Support stretching, https://github.com/flutter/flutter/issues/54059
+// See https://github.com/flutter/flutter/issues/54059
```

Comment:
Is this something we do not want to fix in the future? I took a look at the original issue, I didn't see a concrete suggestion yet. Remove the todo here may make us overlook the issue.

Reply:
I wondered the same, but then I noticed above here on l#337 the same convention was being used. Since there is an active issue I didn't know if this was worth being tallied as part of our tech debt benchmark, when really the intention is to provide context/reference for whomever works on resolving the issue.

Reply:
Sounds good

Reply:
i think if the issue is worth mentioning, and it's an action item that we should definitely take, then it is by definition a TODO. I wouldn't worry about whether it affects the benchmark or not. Just fiddling the numbers doesn't help concretely. :-)